packages/account-v2/src/constants/paymentMethodsConfig.ts

```diff
@@ -164,4 +164,4 @@ export const getPaymentMethodIcon: TPaymentMethodIcon = {
         dark: DerivLightWalletIcon,
         light: DerivLightWalletIcon,
     },
-};
+});
```

Comment:
maybe we can use 'as const' instead of creation new types on line 100?

Reply:
I tried this. But I got the error - `Exported variable 'getPaymentMethodIcon' has or is using name 'SVGRProps' from external module`